docs/source/cluster_management/sweep/targeted-sweep.rst

```diff
@@ -86,15 +86,26 @@ the immutable timestamp lock.
 CONSERVATIVE to THOROUGH
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning::
+
+   You should not change user transactions from read-only to read-write and the sweep strategy from conservative to
+   thorough in the same release. In this case, read-only transactions on old nodes may fail during any state where
+   both old and new nodes are simultaneously operating (though there are no correctness implications).
+
 Thus, a way of changing sweep strategy from CONSERVATIVE to THOROUGH while avoiding downtime is as follows:
-  1. Roll service nodes from a version with CONSERVATIVE sweep strategy to one with THOROUGH_MIGRATION.
-  2. Roll service nodes from a version with THOROUGH_MIGRATION sweep strategy to one with THOROUGH.
+  1. Roll service nodes from a version which uses read-only transactions (e.g. ``runTaskReadOnly``) to one that uses
+     only read-write transactions (e.g. ``runTaskReadWrite``). Both versions should still use the CONSERVATIVE sweep
+     strategy.
+  2. Roll service nodes from a version that uses CONSERVATIVE sweep strategy AND only uses read-write transactions to
+     one with THOROUGH_MIGRATION.
+  3. Roll service nodes from a version with THOROUGH_MIGRATION sweep strategy to one with THOROUGH.
      During this roll, the queue to which cell references are enqueued will vary depending on the individual node.
```

Comment:
You could mention schema bumps here as well?

Reply:
discussed offline: this is how it could be used, but at this level we don't really have the concept of schema version I think